python/cuml/common/logger.pyx

```diff
@@ -73,6 +77,15 @@ level_critical = CUML_LEVEL_CRITICAL
 """Disables all log messages"""
 level_off = CUML_LEVEL_OFF
 
+cdef void _log_callback(int lvl, const char * msg) with gil:
```

Comment:
Even for internal functions, docstrings would be helpful here.

Reply:
Fixed in latest version.